src/plugins/autologin/MQ2AutoLogin.cpp

```diff
@@ -38,9 +38,11 @@ PreSetup("MQ2AutoLogin");
 
 constexpr int STEP_DELAY = 1000;
 
-fs::path CustomIni;
+std::string DBPath = (fs::path(mq::gPathConfig) / "login.db").string();
```

Comment:
Do we ever re-evaluate this? can gPathConfig change?

Reply:
hm, my assumption has always been that this cannot change because all plugins treat it as static as part of the plugin API. I think if this mutates, there will be lots of issues outside of this